Open Judge System/Services/OJS.Services/Cache/Statistics/SubmissionStatisticsCacheService.cs

```diff
@@ -13,15 +12,15 @@
 
     public class SubmissionStatisticsCacheService : ISubmissionStatisticsCacheService
     {
-        private readonly ICacheService cache;
         private readonly ISubmissionsDataService submissionsData;
+        private readonly IRedisCacheService redisCacheService;
 
         public SubmissionStatisticsCacheService(
-            ICacheService cache,
```

Comment:
What happens with this ICacheService? Is it used anywhere? If not it should be deleted